docs/HOWTO_CODE_REVIEW.md

```diff
@@ -1,15 +1,15 @@
 # How to Code Review
-Thank you for stepping up to code review changes for Spoke! Check out [our developer docs](https://github.com/MoveOnOrg/Spoke#more-documentation) for more information about how we write code.
+Thank you for stepping up to code review changes for Spoke! Check out [our developer docs](https://github.com/MoveOnOrg/Spoke/tree/main/docs) for more information about how we write code.
```

Comment:
Let's use `EXPLANATION-development-guidelines.md` even if it's a little messy of a doc

Reply:
Agreed. The idea on this one is to replace it with a link to the developer section in the microsite when we have the URL.